src/assets/assets.controller.ts

```diff
@@ -142,4 +150,30 @@ export class AssetsController {
 
     res.json({ assets });
   }
+
+  @ApiExcludeEndpoint()
+  @Post('update-verified')
```

Comment:
What do you think about making this `@Post('verified')`? It's more inline with RESTful API guidelines

Reply:
`@Post('verify')`?